tests/search/base-searchpage.test.js

```diff
@@ -26,7 +27,8 @@ test('Do search query', async t => {
   await t.wait(1000); // for load
 
   await t.expect(search.inputSearch.exists).ok();
-  await t.expect(search.btnGo.exists).ok();
+  //await t.expect(search.btnGo.exists).ok();
```

Comment:
@dualcnhq once we merge this, can you make sure the commented out tests in this file are live & running?